README.md

```diff
@@ -66,29 +66,17 @@ To build the production Docker image in a local Docker daemon run `sbt docker:pu
 
 ## Configuration
 
-In order to configure Cluster Broccoli, you can add key value pairs to your [configuration](https://www.playframework.com/documentation/2.4.x/Configuration).
+In order to configure Cluster Broccoli, you can add key value pairs to your [configuration](https://www.playframework.com/documentation/2.5.x/Configuration).
 The following configuration properties are supported. Please refer to the [Wiki](https://github.com/FRosner/cluster-broccoli/wiki) for more information about the individual topics.
 
+See [Play configuration](https://www.playframework.com/documentation/2.5.x/Configuration) and [Broccoli's reference.conf](https://github.com/FRosner/cluster-broccoli/blob/master/server/src/main/resources/reference.conf) for available parameters and their meanings.
+
 ### Nomad and Consul
 
 | Property | Description | Default |
 | -------- | ----------- | ------- |
-| `broccoli.nomad.url` | Address of your nomad server | `http://localhost:4646` |
 | `broccoli.consul.url` | Address of your consul server | `http://localhost:8500` |
```

Comment:
Why did you leave these config parameters here? Can't we make them part of the `reference.conf` as well?